wp-content/plugins/core/src/Routes/Routes_Definer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Defines all regular and REST routes to register.
+ */
+declare( strict_types=1 );
+
+namespace Tribe\Project\Routes;
+
+use DI;
+use Tribe\Libs\Container\Definer_Interface;
+use Tribe\Libs\Routes\Route_Definer;
+
+/**
+ * Class to define routes to register.
+ */
+class Routes_Definer implements Definer_Interface {
+	public function define() : array {
```

Comment:
Later on PHPCS rules will come in to make sure that the `:` are touching the method. If you wouldn't mind updating all your methods to remove the space, that would be much appreciated.